mungegithub/options/options.go

```diff
@@ -456,12 +457,29 @@ func toString(optType optionType, val interface{}) string {
 	}
 }
 
+func (o *Options) keysSortedAndWidth() ([]string, int) {
+	keys := make([]string, len(o.options))
```

Comment:
Why do multiple memory allocations when we know the exact size of the array that is needed to back the slice?